### DISCUSSION TITLE
现在kbe引擎自带python3.7，那我怎么才能安装其它python的包呢？

### DISCUSSION THREAD

**Discussion by easeliu** [kbengine/kbengine#1406] [General]
请教一下大家：现在kbe引擎自带python3.7，那我怎么才能安装其它python的包呢？比如tornado或其它异步库？

**Comment by easeliu** [comment]
问题在群友解答后已解决，感谢@kuzhou,@O知足O,@首席村长

**Reply by shangdibaozi** [reply]
所以到底怎么弄的呢？

**Reply by kibozh** [reply]
放到编译后的python的相关包下就ok

**Reply by kbengine** [reply]
只能安装对应版本支持的包，除非你升级引擎集成的python

**Reply by kibozh** [reply]
安装其他python库：https://github.com/kbengine/kbengine/wiki/%E5%AE%89%E8%A3%85%E7%AC%AC%E4%B8%89%E6%96%B9python%E4%BE%9D%E8%B5%96%E5%8C%85%E8%AF%A6%E7%BB%86%E6%95%99%E7%A8%8B
